lib/mongo/server/app_metadata.rb

```diff
@@ -151,10 +157,6 @@ def full_client_document
         end
       end
 
-      def serialize
-        Protocol::Query.new(Database::ADMIN, Database::COMMAND, document, :limit => -1).serialize
-      end
-
       def document
```

Comment:
Can this method please gain either a docstring or a comment providing a summary of what it returns?

Reply:
Documenting comment added.